arviz/plots/ecdfplot.py

```diff
@@ -0,0 +1,107 @@
+"""Plot ecdf and ecdf-difference plot with confidence bands."""
+import numpy as np
+
+from arviz.rcparams import rcParams
+from arviz.plots.plot_utils import get_plotting_function
+
+
+def plot_ecdf(
+    values,
+    values2=None,
+    distribution=None,
```

Comment:
Since this is expected to be a CDF, what about naming this something like `cdf` or `cdf_compare`?

Reply:
I think plot_ecdf is good as we are using the ecdf values to compare, not sure though.

Reply:
I mean the highlighted line, not the function name. Should we rename `distribution` to `cdf`?

Reply:
yeah that makes sense, thanks